app/models/comment.rb

```diff
@@ -0,0 +1,16 @@
+class Comment < ApplicationRecord
+  belongs_to :user, foreign_key: 'AuthorId'
+  belongs_to :post, foreign_key: 'PostId'
+
+  def update_comment_counter(post)
+    comment_count = Comment.where(PostId: post.id).count
+    post.update(CommmentsCounter: comment_count)
+  end
```

Comment:
[ OPTIONAL] 
- You could apply same suggestion I made for Post model post counter method here.